recipes/quickfast/all/conanfile.py

```diff
@@ -37,7 +38,13 @@ def _configure_cmake(self):
 
     def source(self):
         tools.get(**self.conan_data["sources"][self.version])
-        os.rename(self.name + "-" + self.version.replace(".", "_"), self._source_subfolder)
+        extracted_dir = glob.glob("CppCommon-*")[0]
+        os.rename(extracted_dir, self._source_subfolder)
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        extracted_dir = glob.glob("quickfast-*")[0]
```

Comment:
Looks like a copy past mistake 🙊 

```suggestion
        extracted_dir = glob.glob("quickfast-*")[0]
```

Reply:
XD